src/sentry/api/endpoints/release_deploys.py

```diff
@@ -136,6 +136,22 @@ def post(self, request, organization, version):
                     date_started=result.get('dateStarted'),
                 )
 
+            for project in release.projects.all():
+                activity = Activity.objects.create(
+                    type=Activity.DEPLOY,
+                    project=project,
+                    ident=release.version,
+                    data={
+                        'version': release.version,
+                        'deploy_id': deploy.id,
+                        'environment': env.name
+                    },
+                    datetime=deploy.date_finished,
+                )
+            # Somewhat hacky, only send notification for one
+            # Deploy Activity record because it will cover all projects
+            activity.send_notification()
```

Comment:
Not sure if this is realistic, but technically you could get a `NameError` here if there were somehow no projects. Not sure if that's practical to happen or not, but might be safe to just avoid it by setting `activity = None` first, then checking `if activity is not None` here. I don't _think_ there's anything to actually guarantee that we won't have 0 projects here.